challenge-02/README.md

```diff
@@ -58,8 +72,33 @@ Crie uma função com as seguintes características:
 5. Se nenhum argumento for passado, retorne o valor booleano `false`.
 6. E ainda, se nenhuma das condições acima forem atendidas, retorne `null`.
 */
-?
+
+function	recebeargumentos( a , b , c ) {
+	if( a !== undefined && b === undefined && c === undefined ) {
+		return a;
+	} else if( b !== undefined && a === undefined && c === undefined ) {
```

Comment:
-1
Não é necessário fazer todas essas 3 verificações para cada argumento.
Imagine o seguinte: se o segundo argumento for `undefined`, então o terceiro também será (a não ser que o valor `undefined` seja passado explicitamente, o que não é o nosso caso).

Ficou claro?